src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -12,6 +12,177 @@ namespace System.Text.Json.Tests
+            yield return new object[] { "\"1997-07-16T19200\"" };
+
+            // Invalid YYYY-MM-DDThh:mm
+            yield return new object[] { "\"1997-07-16T1\"" };
+            yield return new object[] { "\"1997-07-16Ta0:00\"" };
+            yield return new object[] { "\"1997-07-16T19: 20:30\"" };
+            yield return new object[] { "\"1997-07-16 19:20:30\"" };
+            yield return new object[] { "\"1997-07-16T19:2030\"" };
+
+            // Invalid YYYY-MM-DDThh:mm:ss
+            yield return new object[] { "\"1997-07-16T19:20:3a\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30a\"" };
+            yield return new object[] { "\"1997-07-16T19:20:3045\"" };
+            yield return new object[] { "\"1997-07-16T19:20:304555555\"" };
+
+            // Invalid fractions.
+            yield return new object[] { "\"1997-07-16T19:20:30,45\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30 .\"" };
+            yield return new object[] { "\"abc1997-07-16T19:20:30.000\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+\"" };
+
+            // Invalid TZD.
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+-Z\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555Z \"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+01Z\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+01:\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555 +01:00\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+01:\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555- 01:00\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+04 :30\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555-04: 30\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+0100 \"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555-010\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+430\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555--0430\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+0\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555-01005\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+14:00a\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555-14:0\"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555-14:00 \"" };
+            yield return new object[] { "\"1997-07-16T19:20:30.4555555+14 00\"" };
+
+            // Proper format but invalid time fields
+            yield return new object[] { "\"1997-00-16T19:20:30.4555555\"" };
```

Comment:
I am clearly missing something, why is this invalid? If its **only** the month (00), the comment and test case mismatch (since comment says invalid time fields)

Reply:
Comment mismatch, changed to "Proper format but invalid calendar date, time, or time zone designator fields"